spectral/src/functions/allowPrometheusPlainText.ts

```diff
@@ -0,0 +1,6 @@
+// import { IFunctionResult } from "@stoplight/spectral-core";
```

Comment:
See: https://github.com/bf2fc6cc711aee1a0c2a/architecture/pull/64#discussion_r914730746

Reply:
yea, I'm working on introducing a tag to KFM control plane then using that tag to identify that response and work from that, it would only be on the 200 response. that what I'm going at now